### PR TITLE
[web] Polish web embedding docs.

### PR DESCRIPTION
**Pull request by ditman**
During an internal walkthrough of the Web embedding documents, we came up with the following action items:

* Document the need for the `runWidget` function
* Rename "Full-screen" to "Full page"
* Tweak the title to be smarter, _"embedding"_ is not very meaningful

One of our users was also running the docs, and encountered some of the issues that this PR addresses:

* Fixes: https://github.com/flutter/flutter/issues/153198

There's also some minor typo/formatting fixes.

## Presubmit checklist

- [ ] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/content/platform-integration/web/embedding-flutter-web.md
Comment (ditman) [inline src/content/platform-integration/web/embedding-flutter-web.md:3]: Please, @kevmoo, take a look at the changes to the title and left navbar; here's what they look like with this PR:

Screenshot 2024-09-12 at 6 31 32 PM

They're the smartest that I can come up with 😛

**Comment by ditman** [comment]
Thanks robot, here's the actual page that I'm touching:

* https://flutter-docs-prod--pr11133-web-embedding-user-journey-v0hpre0e.web.app/platform-integration/web/embedding-flutter-web

**Review by sfshaza2 — APPROVED** [review]
LGTM.  Thanks so much, @ditman! Will wait for @kevmoo's review before landing.

**Review by kevmoo — APPROVED** [review]
🙏

**Comment by ditman** [comment]
If both @sfshaza2 and @kevmoo are happy, I'm clicking the "Squash and merge" button right now.

Thanks for the quick review!

**Pull request merged**
